Willning/DLOplanTool
# Dlo 20rework

### **Related Issue/Keyword:**
 Mapping Screen prototype completion. 
### **Description:**
Yeah, this the biggie. Project mapping and saving working. Can now create a mapping with a medium restricting modes for that plan. Also loading from projects and recent projects work. 

### **Testing:**
Make a plan and save, should load. 

**Important note** 
Any previous data in the DB will be deprecated and won't load properly due to format changes. 

**Steps for manual testing:**

1. Make a plan and add mappings etc. 
2. Save
3. Load plan and everything should be there. 

### **Checklist:**
